### PR TITLE
Contributing

### PR DESCRIPTION
**Pull request by cliffcaseyyet**
Contribution Agreement, Code of Conduct, links from README.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+## Code of Conduct
+
+The Yet Analytics Open Source Community has a [Code of Conduct](CODE_OF_CONDUCT.md) which should be read and followed when contributing in any way.
+
+## Issue Reporting
+
+Yet Analytics encourages users to contribute by reporting any issues or enhancement suggestions via [GitHub Issues](https://github.com/yetanalytics/lrsql/issues).
+
+Before submission, we encourage you to read through the existing [Documentation](docs/index.md) to ensure that the issue has not been addressed or explained.
```
Comment (kelvinqian00) [inline CONTRIBUTING.md:17]: The Documentation link here 404's.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+individual, or aggression toward or disparagement of classes of individuals.
+
+**Consequence**: A permanent ban from any sort of public interaction within
+the community.
+
+## Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant][homepage],
+version 2.1, available at
+[https://www.contributor-covenant.org/version/2/1/code_of_conduct.html][v2.1].
```
Comment (kelvinqian00) [inline CODE_OF_CONDUCT.md:119]: This sentence displays with two links to the same page - we can get away with just one.

**Review by kelvinqian00 — APPROVED** [review]

**Review by blakeplock — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ A SQL-based Learning Record Store.
 
 A Learning Record Store (LRS) is a persistent store for xAPI statements and associated attachments and documents. The full LRS specification can be found in Part 3 of the [xAPI specification](https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-Communication.md). SQL LRS is distinct from other LRSs developed at Yet Analytics for being SQL-based and supporting multiple SQL database management systems (DBMSs) like H2, SQLite, and Postgres.
 
-## Releases
```
Comment (milt) [inline README.md:14]: Haha my editor kept wanting to do this too

**Review by milt — APPROVED** [review]
I've read through, looks good to me

**Pull request merged**